dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/RequestPasswordResetEmailTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\Customer;
+
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+
+class RequestPasswordResetEmailTest extends GraphQlAbstract
```

Comment:
Please add scenario for locked customer.

Reply:
**Done**